### PR TITLE
Test: Delete function from the setup that is not longer used and add …

### PR DESCRIPTION
**Pull request by GabyCT**
…--rm flag

- This will delete the use of the function of clean_docker_ps in the setup as
this one is no longer used in our test-common.bash
- Add the flag of --rm when running a container

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Comment by chavafg** [comment]
qa-failed

**Comment by devimc** [comment]
lgtm

**Comment by chavafg** [comment]
still having some issues with docker swarm, not related to this change, moving to qa-passed

**Comment by chavafg** [comment]
qa-passed

**Comment by jodh-intel** [comment]
HI @GabyCT - good catch. It looks like c8b933adb193d611fc0998368f866d8028efe99a removed the `clean_docker_ps` function. But there still other files calling this function:

```
$ grep -rlE "\<clean_docker_ps\>" | wc -l
7
```

**Comment by GabyCT** [comment]
@jodh-intel, thanks for the observation. I noticed that the other files that are calling the function are part of the metrics as the integration docker tests are now fixed.

File: tests/integration/docker_images/popular-containers-images.bats [hunk 1]
```diff
@@ -289,7 +290,7 @@ setup() {
 }
 
 @test "start instance in percona container" {
-	if $DOCKER_EXE run -i percona perl -e 'print "Clear Containers\n"' | grep LANG; then false; else true; fi
+	if $DOCKER_EXE run --rm -i percona perl -e 'print "Clear Containers\n"' | grep LANG; then false; else true; fi
```
Comment (dlespiau) [inline tests/integration/docker_images/popular-containers-images.bats]: This construct "if xxx | grep foo; then false; else true; fi" could have been done with "if xxx | grep -v foo" (no need to do anything about this, for next time you're around that code).

**Comment by dlespiau** [comment]
lgtm, but those remaining occurrences of clean_docker_ps will have to be removed. Filed https://github.com/01org/cc-oci-runtime/issues/652.

**Pull request merged**